docs/about/neon_ecosystem.md

```diff
@@ -47,42 +49,26 @@ The Neon Proxy Operator accepts payment from the user in [NEON tokens](/docs/tok
 
 Owners of the Neon token may engage in the Neon DAO activities.
 
-### Neon Operators
+## Neon DAO
 
-**Neon Operators** run Neon Proxy servers to facilitate the seamless execution of Ethereum-like transactions on Solana. It is this proxy service that allows Ethereum dApps to be ported to Neon with no code changes. Operators accept payment in NEON and make payments within the Solana ecosystem in SOL.
+The DAO essentially owns the Neon EVM and is also a grant-issuing authority able to direct future research and development. The Neon DAO exists as a series of contracts deployed on Solana to provide a Layer 0 (L0: a social/governance layer) to control the functionality of Neon EVM. The DAO provides a web interface to allow the community, i.e. NEON holders, to raise and vote on proposals that impact the functioning of the Neon EVM.
 
 ## Neon Tools
 
-### Explorers
-
-Two explorers are available:
-
-#### 1. NeonScan
+### Block explorer
 
 [NeonScan](https://neonscan.org/) provides a block explorer and analytics platform to examine Neon transactions within Solana.
 
-#### 2. NeonAnalytics
-
-[NeonAnalytics](https://neon.aleph.cloud/) provides a block explorer providing a high-level overview of various metrics. It allows users to analyze deployed contracts.
-
-### Tracing API
-
-More sophisticated than NeonScan, Neon's Tracing API is an extension for Neon Proxy. It assists developers to test, debug, and understand the behavior of their smart contracts on Neon EVM. The API provides a full externality trace on any Neon transactions executed on the Solana blockchain. 
-
-The Tracing API supports a rerun of historical transactions, runs transactions on historical states, and analyzes the state of the Solana blockchain after the execution of each instruction. 
-
-> More details are available in [our blog](https://medium.com/neon-labs/neon-proxy-tracing-api-fdb3842a80fa).
-
 
 ### NeonPass
 
-[NeonPass](https://devnet.neonpass.live/), currently available for devnet, transfers ERC-20 tokens between Solana and Neon EVM. The [ERC-20 SPL wrapper](/docs/developing/deploy_facilities/interacting_with_spl_tokens) makes such transfers possible via (non-custodial) wallets.
+[NeonPass](https://devnet.neonpass.live/), currently available for DevNet, transfers ERC-20 tokens between Solana and Neon EVM. The [ERC-20 SPL wrapper](/docs/developing/deploy_facilities/interacting_with_spl_tokens) makes such transfers possible via (non-custodial) wallets.
```

Comment:
I'd prefer to remove DevNet mentioning, cause we'll forget to remove it once we be in Mainnet